### PR TITLE
Blog: Refresh Moving from open source Elasticsearch to OpenSearch

### LINKED ISSUE #1135: Blog Refresh: Moving from open source Elasticsearch to OpenSearch
Describe the blog post your would like to write
This is a blog refresh to update technical information as of OpenSearch 2.4; blog link https://opensearch.org/blog/technical-posts/2021/10/moving-from-opensource-elasticsearch-to-opensearch/

What is the title of the blog post?
Moving from open source Elasticsearch to OpenSearch

Who are the authors?
Sarat Vemulapalli

What is the proposed posting date?
11/10

### PR DESCRIPTION
**Pull request by vagimeli**
Signed-off-by: vagimeli <vagimeli@amazon.com>

Describe the blog post your would like to write
This is a blog refresh to update technical information as of OpenSearch 2.4; blog link https://opensearch.org/blog/technical-posts/2021/10/moving-from-opensource-elasticsearch-to-opensearch/

What is the title of the blog post?
Moving from open source Elasticsearch to OpenSearch

Who are the authors?
Sarat Vemulapalli

What is the proposed posting date?
11/10
 
### Issues Resolved
#1135 

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by vagimeli** [comment]
Hi @saratvemulapalli, This PR is ready for your review. Please confirm technical accuracy and provide your feedback.

File: _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown [hunk 1]
```diff
@@ -50,8 +51,11 @@ POST /_reindex
...
 }
 ```
+Lastly, download the latest version of OpenSearch from the [OpenSearch downloads page](https://opensearch.org/downloads.html). If you need help with installation, see the guidance in [Install and Configure OpenSearch](https://opensearch.org/docs/latest/opensearch/install/index/).
+
+As described in a [previous blog post](https://opensearch.org/blog/technical-posts/2021/07/how-to-upgrade-from-opendistro-to-opensearch/), open-source Elasticsearch is the foundation for Open Distro, and it has the same upgrade paths. You can follow through the steps for the path you choose to upgrade your cluster.
 
-### Support Matrix for open source Elasticsearch to OpenSearch
+## Support matrix for open-source Elasticsearch to OpenSearch
 
 |Open Source Elasticsearch|Recommended Upgrade Path	|
```
Comment (vagimeli) [inline _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown:60]: @saratvemulapalli Do we have any updates to make to this table?

File: _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown [hunk 2]
```diff
@@ -50,8 +51,11 @@ POST /_reindex
    }
 }
 ```
+Lastly, download the latest version of OpenSearch from the [OpenSearch downloads page](https://opensearch.org/downloads.html). If you need help with installation, see the guidance in [Install and Configure OpenSearch](https://opensearch.org/docs/latest/opensearch/install/index/).
+
+As described in a [previous blog post](https://opensearch.org/blog/technical-posts/2021/07/how-to-upgrade-from-opendistro-to-opensearch/), open-source Elasticsearch is the foundation for Open Distro, and it has the same upgrade paths. You can follow through the steps for the path you choose to upgrade your cluster.
```
Comment (vagimeli) [inline _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown:56]: @saratvemulapalli Is it still relevant to link to this blog? Is there an updated resource we want to include instead?

File: _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown (hunk 1 above)
Comment (saratvemulapalli) [reply _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown:60]: @vagimeli this doesnt change. It should be good.

File: _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown (hunk 2 above)
Comment (saratvemulapalli) [reply _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown:56]: @vagimeli definitely. Looking at the blog it goes in details of how each type of upgrade should be performed irrelevant of versions/Elasticsearch/OpenSearch.

File: _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown [hunk 3]
```diff
@@ -7,37 +7,38 @@ date: 2021-10-29
...
 
-Static settings defined in `elasticsearch.yml` / `opensearch.yml` are not moved automatically during an upgrade.
-OpenSearch 1.1.0 is bundled with a new [upgrade tool](https://opensearch.org/docs/latest/upgrade-to/upgrade-to/#upgrade-tool) which connects to an Elasticsearch cluster and moves static settings which are defined in `elasticsearch.yml` to `opensearch.yml` automatically.
+* Cluster settings: All dynamic settings that are supported in the new cluster will be migrated, and the ones that were deprecated will be archived.
+* Indexes: All compatible indexes (that is, all your data) will be readable/writable by the new software version.
 
-Yay! Now you know what an upgrade is, lets buckle up for the upgrade.
+Static settings defined in the  `opensearch.yml` or `elasticsearch.yml` files are not moved automatically during an upgrade. Each version of OpenSearch is bundled with the [`opensearch-upgrade`](https://opensearch.org/docs/latest/upgrade-to/upgrade-to/#upgrade-tool) tool. The tool connects to an Elasticsearch cluster and moves the static settings. The cluster and the settings are defined in the `elasticsearch.yml` to `opensearch.yml` files automatically.
```
Comment (saratvemulapalli) [inline _posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown:26]: Also the upgrade tool, migrates the keystore secrets to opensearch if the customer has any. We could add that as well.

**Review by saratvemulapalli — DISMISSED** [review]
@vagimeli overall the changes look good to me.
Thanks for taking a stab at this.

**Review by saratvemulapalli — APPROVED** [review]

**Review by krisfreedain — APPROVED** [review]
spoke with @pajuric - approved - pushing this out. thanks @vagimeli

**Pull request merged**